layouts/partials/sidebar.html

```diff
@@ -8,13 +8,12 @@ <h2>{{ .Site.Params.Author }}</h2>
     <a href='{{ absLangURL "about/" }}' class="button mt-1" role="button">{{ i18n "read_more" }}</a>
     <h2 class="mt-4">{{ i18n "featured_posts" }}</h2>
     <ul>
-    {{ range where .Site.RegularPages "Params.featured" true }}
-     {{- if ne $title .Title }}
```

Comment:
need this check `{{- if ne $title .Title }}`, see conversation in #19

Reply:
Current viewing post is filtered by `"File.UniqueID" "!=" .File.UniqueID` already.